Makefile

```diff
@@ -10,7 +10,20 @@ DATE := $(shell date +%Y-%m-%d)
 export DATE
 
 # CONTAINER_RUNTIME defines the container runtime to use for building the bundle image.
-CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
+.PHONY: check-container-runtime
+check-container-runtime:
+	@if podman ps >/dev/null 2>&1; then \
+		echo "Using podman as container runtime"; \
+		echo podman > .container_runtime; \
```

Comment:
maybe we don't need the `.container_runtime` file, as this will be checked in dev systems. WDYT?

Reply:
Thank you. Very good catch! 
Fixed in https://github.com/perses/perses-operator/pull/55/commits/3f17c3da11a6da3ecaef804811ad9a7acccd42cf commit.